playbooks/common/openshift-cluster/upgrades/etcd/backup.yml

```diff
@@ -29,12 +29,18 @@
   - name: Check available disk space for etcd backup
     shell: df --output=avail -k {{ openshift.common.data_dir }} | tail -n 1
     register: avail_disk
+    # AUDIT:changed_when: `false` because we are only inspecting
```

Comment:
Are these Audit comments meant to be temporary? If they are meant to be permanent, I think I would prefer something more concise, like flake8/pylint/ansible-lint supports.

I also wonder if we can leverage a tool like [ansible-review](https://github.com/willthames/ansible-review) to automate some of this enforcement.